SCClassLibrary/Common/Streams/Patterns.sc

```diff
@@ -673,7 +673,7 @@ Pkey : Pattern {
 
 	embedInStream { |inval|
 		var outval, keystream = key.asStream;
-		repeats.value(inval).do {
+		(repeats.value(inval) ?? { inf }).do {
```

Comment:
I've explained my reasoning in the main thread -- in short, this looks innocent enough, but it sets a precedent for a large semantic change for patterns. I don't think we should do it casually, no matter how convenient it might be in this one case.

Reply:
I'm pretty sure we already had this conversation. I believe it's categorically impossible for a push + method call every time (?) to be faster than a jump + only sometimes push (?? {}). I'll benchmark again when I'm at my desk but I'm pretty sure it will confirm.

Reply:
Results just posted to the mailing list. `??` is definitely faster. You might have had an outlier result once upon a time, but it doesn't hold up to repeated tests.